### PR TITLE
Jshint fixes

### PR DESCRIPTION
**Pull request by tpkelly**

### CODE REVIEW

**Comment by tpkelly** [comment]
Not ready to go until appveyor fails a future commit

File: VotingApplication/VotingApplication.Web/Scripts/Controllers/ResultsPageController.js [hunk 1]
```diff
@@ -127,6 +130,8 @@
             var winningScore = 0;
 
             var datapoints = [];
+            var keys = Object.keys(groupedData);
```
Comment (JDawes-ScottLogic) [inline VotingApplication/VotingApplication.Web/Scripts/Controllers/ResultsPageController.js]: Was this line supposed to be included? I can't see that ```keys``` is used anywhere?

Comment (JDawes-ScottLogic) [inline VotingApplication/VotingApplication.Web/Scripts/Controllers/ResultsPageController.js]: Caused by a merge conflict.

**Comment by JDawes-ScottLogic** [comment]
thumbs-up-bruce-lee_2

**Comment by stevenhillcox** [comment]
Why is this still still considered 'open' ?

**Pull request merged**